src/components/structures/FilePanel.tsx

```diff
@@ -265,7 +270,7 @@ class FilePanel extends React.Component<IProps, IState> {
             />
         );
 
-        const isRoomEncrypted = this.noRoom ? false : MatrixClientPeg.safeGet().isRoomEncrypted(this.props.roomId);
+        const isRoomEncrypted = this.noRoom ? false : this.state.isRoomEncrypted;
```

Comment:
This seems to have a risk of being racy, would we not need an interstitial loading state while we figure out if the room is encrypted? Otherwise the search warning may flicker in